src/components/LoanCards/KvClassicLoanCardContainer.vue

```diff
@@ -435,7 +435,8 @@ export default {
 			}
 		},
 		getTargetPosition() {
-			const target = document.getElementById('basket-exp');
+			const targets = [...document.querySelectorAll('[data-testid="header-basket"]')];
```

Comment:
It looks like this will be good code to abstract out when we port this over to CPS, so we don't have duplication.